File: 프로그래머스/3/12938. 최고의 집합/최고의 집합.py
def solution(n, s):
    if n>s:
        return [-1]
    elif n==s:
        return n
    answer = []
    for i in range(n-(s%n)):
        answer.append(s//n)
    for i in range(s%n):
        answer.append((s//n)+1)
    
    return answer

File: 프로그래머스/3/12938. 최고의 집합/최고의 집합.js
function solution(n, s) {
    if(n>s) return [-1];
    var answer = [];
    
    let first = Math.floor(s/n)
    for(let i=0;i<n;i++){
        answer.push(first)
    }
    
    let idx = 0
    let second = s%n
    
    while(second!==0){
        answer[idx] +=1
        second--;
        idx++;
    }
    
    return answer.sort();
}